cmd/agent/main.go

```diff
@@ -25,7 +27,20 @@ var (
 )
 
 func main() {
-	appmain.Run("Juice Agent", build.Version, func(group task.Group) error {
+	name := "Juice Agent"
+	config := appmain.Config{
+		Name:    name,
+		Version: build.Version,
+
+		SentryConfig: sentry.ClientOptions{
+			Release:          fmt.Sprintf("%s@%s", name, build.Version),
+			Dsn:              "https://4bd058ff6346e984d53edcba7c22ea63@o4505739073486848.ingest.sentry.io/4505763525427200",
```

Comment:
Do we want this hardcoded in the repo? Maybe an env variable during development?

Reply:
you could put this in `.env` - it should get loaded on startup into env